en/docs/install-and-setup/setup/security/logins-and-passwords/harshicrop-vault-extension.md

```diff
@@ -0,0 +1,298 @@
+
+# Integrate with HashiCorp Vault
+
+Using [HashiCorp Vault extension](https://github.com/wso2-extensions/carbon-securevault-hashicorp/tree/master), you can set up HashiCorp Vault to store passwords that are mapped to aliases instead of the actual passwords.
+
+## Setting up using root token authentication
+
+### Step 1: Setup HashiCorp Vault
```

Comment:
```suggestion
### Step 1 - Setup HashiCorp Vault
```

Reply:
corrected in 1d8bdffa01d44e5da79703dd2060f0a621bb659a